src/clients/c/tb_client_header_generate.zig

```diff
@@ -137,7 +138,9 @@ fn emit_struct(
     try buffer.writer().print("}} {s};\n\n", .{c_name});
 }
 
-pub fn main() !void {
+// This tests generates the `tb_client.h` file. If you make changes to the C ABI of the client, run
+// `./zig/zig build test -- "tb_client.h: generate"` to re-generate the header file.
+test "tb_client.h: generate" {
```

Comment:
I am slightly concerned here that the same CU both generates the header file, and also `@cImports` it, so there _might_ be some unfortunate cases where we cant' compile tests because the header is broken, but we can't renegenerate header because we can't compile tests. I reasoned that that's OK, as you can always comment out the offending code to unblock re-generation, which is, yes, non-ideal, but I think is fine for the rare use-case.

But then I recalled that Zig is lazy, and that, if you pass in `./zig/zig build test -- "tb_client.h: generate"`, then _only_ this specific test will get compiled. So we are fine here actually!